control-plane/pkg/keda/keda.go

```diff
@@ -83,10 +90,36 @@ func GenerateScaleTriggers(cg *kafkainternals.ConsumerGroup, triggerAuthenticati
 	return triggers, nil
 }
 
-func GenerateTriggerAuthentication(cg *kafkainternals.ConsumerGroup, saslType *string) (*kedav1alpha1.TriggerAuthentication, *corev1.Secret, error) {
+func GenerateTriggerAuthentication(cg *kafkainternals.ConsumerGroup, saslType *string, protocol *string, caCert *string) (*kedav1alpha1.TriggerAuthentication, *corev1.Secret, error) {
```

Comment:
should we do a struct instead of a bunch of new args here?

Reply:
Coincidentally, I need to add a few more args for an update coming up. Will need a struct. Thanks!